src/components/Table/PeopleTable.tsx

```diff
@@ -0,0 +1,81 @@
+import classNames from 'classnames';
```

Comment:
Rename the folder to `PeopleTable` (or vice versa - names should match) and add index file in this folder with re-export `PeopleTable` component

That will improve your files structure
the same for navigation